Documentation/BuildInstructions.md

```diff
@@ -14,7 +14,7 @@ Optional: `fuse2fs` for [building images without root](https://github.com/Sereni
 
 #### GCC 13 or Clang 17+
 
-A host compiler that supports C++23 features is required for building host tools, the newer the better. Tested versions include gcc-13 and Clang 17 through 19.
+A host compiler that supports C++26 features is required for building host tools, the newer the better. Tested versions include gcc-13 and Clang 17 through 19.
```

Comment:
I think this should be bumped to gcc-14, but I haven't tested with that GCC version yet.